### ISSUE TITLE
Can I use SAPI mode cgi-fcgi instead of cli for execution of base.php?

### ISSUE THREAD

**Issue by BerChill** [Alanaktion/phproject#230]
Hi Alan,

my shared hoster uses cpanel and it seems that there is only support for the following PHP Handlers:     cgi, dso, none, fcgi, suphp
see cpanel documentation (https://documentation.cpanel.net/display/CKB/How+to+Configure+PHP+and+suEXEC+from+the+Command+Line)

Cron jobs on my hoster are executed using cgi-fcgi and they told me already that they won't change that. Can I change the required handler for executing base.php (line 8) to cgi-fcgi?

**Comment by Alanaktion** [comment]
I'd prefer not to use `cgi-fcgi`, since that will prevent the standard CLI-based cron jobs from working. Something like this may work for more hosts:
```php
if (PHP_SAPI != 'cli' && !defined('STDIN')) {
    // ...
}
```
If that code works for you, we can add it to the repo.

**Comment by BerChill** [comment]
Hi,

unfortunately this didn't work. From what I've read it might be that with the standard configuration STDIN is never defined when using cgi-fcgi.
However, after some trial and error I found the correct path to make cPanel use the cli interface. Turns out the support of my shared hoster doesn't really know what they are talking about.

I understand that you wouldn't want to make changes that make it more complicated for non-shared-hoster-users. From what I can tell, all the funcionality of phproject can be used with one of the good shared hosters as of now. It would be great if this holds true for future development. 
phproject is truly the best PM system that doesn't require much to set up and run while offering state of the art functionality. IMO its perfect for small organizations that just have tech-savy members but no "real" IT-guy.

Thanks again for your support!
